src/flags.rs

```diff
@@ -117,6 +121,22 @@ impl Default for Flags {
     }
 }
 
+#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+pub enum SizeFlag {
+    Default,
+    Small,
+}
+
+impl<'a> From<&'a str> for SizeFlag {
+    fn from(size: &'a str) -> Self {
+        match size {
+            "size" => SizeFlag::Default,
```

Comment:
What do you think of replacing "size" by something more explicit like "classic" or even "default"?

Reply:
I think we will use `default` and maybe we could display raw number of bytes for `classic`?